README.md

```diff
@@ -113,60 +49,23 @@ namespace Example {
 }
 ```
 
-You can compile it to OpenAPI 3.0 by using the following command:
+Install the dependencies of sample.tsp:
 
 ```
-tsp compile sample.tsp --emit @typespec/openapi3
+tsp install
 ```
 
-Once it compiles, you can find the emitted OpenAPI document in `./tsp-output/openapi.json`.
-
-You can also pass in a directory instead of a file to `tsp compile`. That's
-equivalent to passing `main.tsp` in that directory.
-
-### Formatting TypeSpec files
-
-TypeSpec provides an auto-formatter to keep your specs clean and organized.
-`node_modules` folders are automatically excluded by default
-
-```bash
-tsp format <patterns...>
-
-# Format all the files in the current directory with the TypeSpec extension.
-tsp format **/*.tsp
+Compile it to OpenAPI 3.0:
 
-# Exclude certain patterns. Either use `!` prefix or pass it via the `--exclude` or `-x` option.
-tsp format **/*.tsp "!my-test-folder/**/*"
-tsp format **/*.tsp --exclude "my-test-folder/**/*"
 ```
-
-### Installing VS Code Extension
-
-```
-tsp code install
+tsp compile sample.tsp --emit @typespec/openapi3
 ```
 
-This will download and install the latest VS Code extension. Use `tsp code uninstall` to remove it. Pass `--insiders` if you use VS Code Insiders edition.
-
-If `tsp-server` cannot be found on PATH by VS Code in your setup, you can
-configure its location in VS Code settings. Search for "TypeSpec" in File ->
-Preferences -> Settings, and adjust `typespec.tsp-server.path` accordingly. You may
-need to restart VS Code after changing this. This should be the path to the `@typespec/compiler` package. (e.g. `./node_modules/@typespec/compiler`)
-
-You can also configure a project to use a local npm install of
-`@typespec/compiler`. See [local-typespec sample](packages/samples/local-typespec).
-
-### Installing Visual Studio Extension
-
-```
-tsp vs install
-```
+You can find the emitted OpenAPI output in `./tsp-output/openapi.json`.
 
-This will download and install the latest Visual Studio extension. Use `tsp vs uninstall` to remove it.
+## [Troubleshooting](./troubleshooting.md)
```

Comment:
Headers should not be links, I wonder if we can just drop this entirely (if you are having problems, probably you are consulting the docs site as opposed to the readme?)